install_config/configuring_gce.adoc

```diff
@@ -48,8 +48,9 @@ During xref:../install_config/install/advanced_install.adoc#install-config-insta
 
 [IMPORTANT]
 ====
-When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
+* When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
 parameters must be defined.
+* For running loadbalancer services using the Google compute platform, the nodes (Compute Engine VM instances) must be tagged with `<openshift_gcp_prefix>ocp` (add `ocp` suffix). For example, if the value of parameter `openshift_gcp_prefix` is set to `mycluster`, the nodes must be tagged with `myclusterocp`. See link:https://cloud.google.com/vpc/docs/add-remove-network-tags[Adding and Removing Network Tags] for more information on how to add network tags to Compute Engine VM instances.
```

Comment:
@gaurav-nelson 
s/parameter `openshift_gcp_prefix`/the `openshift_gcp_prefix` parameter